NekoGirlStateMachine.js

```
@@ -2,79 +1,83 @@
-  static HAPPY = 'happy';
-  static NEUTRAL = "neutral"
-  static HAPPY = "happy"
-  static SAD = "sad"
-  static ANGRY = "angry"
-  static SURPRISED = "surprised"
+    static NEUTRAL = "neutral";
+    static HAPPY = "happy";
+    static SAD = "sad";
+    static ANGRY = "angry";
+    static SURPRISED = "surprised";
 }
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)